metrics/web/test/test_utils/metrics_config_stub.dart

```diff
@@ -0,0 +1,24 @@
+import 'package:metrics/common/domain/entities/metrics_config.dart';
+
+/// A stub implementation of the [MetricsConfig] to use in tests.
+class MetricsConfigStub implements MetricsConfig {
+  @override
+  final String googleSignInClientId;
```

Comment:
What do you think if we place googleSignInClientId first in the actual javascript file and in WebMetricsConfig like in this file? While looking at this PR I felt like it's more natural order - as if Sentry fields are placed first they took too much space and reader might think that all of the fields are Sentry related?

![image](https://user-images.githubusercontent.com/442086/104631450-ef0a7f80-56a4-11eb-9f0c-bcca2bf18228.png)